hw/ip/otbn/data/insns.yml

```diff
@@ -162,6 +170,37 @@ encoding-schemes: enc-schemes.yml
 #
 #  bytes:   An integer giving the width of the operation in bytes. This is
 #           required if type is mem-* and cannot be used otherwise.
+#
+#
+# If specified, the iflow field for an instruction should be a list of
+# dictionaries, each specifying an information flow "rule". Rules have the
+# following fields:
+#
+#   to: A list of strings, representing the locations that receive information
+#       from locations in the "from" list (i.e. information-flow sinks). Each
+#       string should be either:
+#       - a register operand name such as "grd" or "wrs1"
+#       - "wref-<reg operand name>" for instructions where a WDR is indirectly
+#         accessed via a GPR operand
+#       - "dmem" for memory, which is treated as a monolithic block
+#       - "acc" or "mod" to represent special registers ACC and MOD
+#       - a flag (represented as "<flag group>-<flag>" e.g. fg0-all, flags-c), where:
+#         * <flag group> can be either "fg0", "fg1", or (if the instruction has
+#           a `flag_group` operand) simply "flags" to select that flag group
+#         * <flag> can be l, c, m, z, or "all".
+#
+#   from: List of strings with the same format as the "to" field; represents
+#         the locations that information in this rule is flowing from (i.e.
+#         information-flow sources).
```

Comment:
Maybe it would be worth explicitly saying what `from: []` means here? (It's a really clever encoding, so I think you should take any credit due!)

Reply:
Speaking of giving credit where credit is due, just going to state for the record that you helped me design the whole information-flow encoding -- thanks!